### ISSUE TITLE
Troubleshoot: ValueError: Element is not a child of this node

### ISSUE THREAD

**Issue by hotgossip1** [inkstitch/inkstitch#2629]
new to inkstitch recieved this error message, svg file included.
![santa v2](https://github.com/inkstitch/inkstitch/assets/153403247/bf346bbf-bca0-4ef6-9f22-b387ba8bf264)

raceback (most recent call last):
  File "inkstitch.py", line 84, in <module>
  File "inkex\base.py", line 250, in run
  File "lib\extensions\troubleshoot.py", line 47, in effect
  File "src\lxml\etree.pyx", line 961, in lxml.etree._Element.remove
ValueError: Element is not a child of this node.

apologies if in wrong place, never done this before and although i have had inkscape/inkstitch for a while this is the first time its thrown up an error message,  hope i have placed this in correct location.

**Comment by bianchifan** [comment]
What system?
What version of Inkscape, Ink/Stitch?

No problems here on Linux with current actual versions.
Maybe you should rework yor design and parametrize it.
What object should be filled and what object may look better with running/satin stitches...

The zip contains some popular formats.
![289329550-bf346bbf-bca0-4ef6-9f22-b387ba8bf264 svg](https://github.com/inkstitch/inkstitch/assets/17491743/aa1ccd39-3f18-40de-a881-c84ef4d92f46)

[289329550-bf346bbf-bca0-4ef6-9f22-b387ba8bf264.zip](https://github.com/inkstitch/inkstitch/files/13627763/289329550-bf346bbf-bca0-4ef6-9f22-b387ba8bf264.zip)

**Comment by lexelby** [comment]
Hi @hotgossip1! Thanks for the bug report.  This definitely shouldn't happen, so it's not something wrong with your SVG.

Did the error day what version of Ink/Stitch you're running?  That would be useful for us.

**Comment by kaalleen** [comment]
Thank you for the error report. The error is still reproducible with v3.0.2 - that if the main group (layer1) is converted to an actual group. If not troubleshoot will do bad positioning of the pointers. So that would be an issue too. Thank you for sending the svg along. This is really helpful.

**Comment by hotgossip1** [comment]
Hello,

Sorry i missed this email, i am using the inkscape-1.3-alpha_2023-04-27_3c4f48da9-x64 master edition as i cannot get the other ones to run on my windows 10 laptop.  Other versions keep freezing and throwing up errors.

I am relatively new to inkscape, and using the program for inkstitch as I wanted to design my own embroidery files.  Being older 68 I'm not sure if i have followed all the steps needed to run the inkscape program onto my laptop as its says on the inkstitch page to do additional steps which I've tried to do... so not sure if its my fault.  It would be helpful if someone at inkscape could add the inkstitch part as a simply install program as I keep getting errors about debugging and its bugging the heck out of me, as I don't know how to fix the issue and don't know how to add programming instructions into the programs as its not clear for a novice .  I followed the additional steps in inkstitch as best I could and also installed visual studio and I still get errors... see attachment.  I hope this helps I have tried removing and reinstalling but I cant seem to crack the problem.

Program works most of the time, but then it just freezes and shuts down.  Any help is gratefully appreciated as I have lost numerous amounts of work and sometimes it doesn't autosave even though I have it set to save.

Kind Regards
Mary
________________________________
